blocks/breadcrumb/breadcrumb.js

```diff
@@ -0,0 +1,45 @@
+const getPageTitle = async (url) => {
+  const resp = await fetch(url);
+  const html = document.createElement('div');
+  html.innerHTML = await resp.text();
+  return html.querySelector('title').innerText;
+};
+
+const getAllPaths = async (paths) => {
+  const result = [];
+  // remove first and last slash characters
+  const pathsList = paths.replace(/^\/|\/$/g, '').split('/');
+  for (let i = 0; i < pathsList.length; i += 1) {
```

Comment:
minor, bit I prefer a `forEach` structure, so `pathsList.forEach((path) => {})`

Reply:
try to use forEach in the beginning, but figure that it is not straight forward to do async await with forEach, so had to do this.
See [Do not use forEach with async-await](https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971)